lib/SetAPI/assembly/AssemblySetInterfaceV1.py

```diff
@@ -0,0 +1,88 @@
+
+
+
+from SetAPI.generic.SetInterfaceV1 import SetInterfaceV1
+
+from biokbase.workspace.client import Workspace
+
+from pprint import pprint
+
+
+
+class AssemblySetInterfaceV1:
+
+
+    def __init__(self, workspace_client):
+        self.ws = workspace_client
+        self.setInterface = SetInterfaceV1(workspace_client)
+
+    def save_assembly_set(self, ctx, params):
+        if 'data' in params:
+            self._validate_assembly_set_data(params['data'])
+        else:
+            raise ValueError('"data" parameter field required to save an AssemblySet')
+
+        save_result = self.setInterface.save_set(
+                'KBaseSets.AssemblySet',
+                ctx['provenance'],
+                params
+            )
+        info = save_result[0]
+        return {
+            'set_ref': str(info[6]) + '/' + str(info[0]) + '/' + str(info[4]),
+            'set_info': info
+        }
+
+
+    def _validate_assembly_set_data(self, data):
+        # TODO: add checks that only one copy of each assembly data is in the set
```

Comment:
I know this didn't exist yet for ReadsSet, but we should probably implement this to prevent duplicate entries.  We should be able to do this in the generic Set code.